New data for individual Indiana counties

migurski opened the pull request:
Added so far:

- [x] Allen: https://gis.acimap.us/projectservicesadmin/rest/services/Accela/Accela_Production/MapServer/51
- [x] Hamilton: http://gis.hamiltoncounty.in.gov/ArcGIS/rest/services/Precincts/MapServer/0
- [x] Elkhart: https://elb2.39dn.com/arcgis/rest/services/eGISDynamicServices/ElkhartINDynamic/MapServer/103
- [x] St. Joseph: http://stjosephcountyindiana.com/788/Layer-List
- [x] Document sources in Indiana README

To do:

- [ ] More from largest-population counties at https://en.wikipedia.org/wiki/List_of_counties_in_Indiana

nvkelso commented on Makefile:
```diff
@@ -463,22 +463,34 @@ out/17-illinois/state.gpkg: data/17-illinois/statewide/2010/tl_2012_17_vtd10.zip
 	# GPKG are weird
 	rm -f $@
-	ogr2ogr -sql "SELECT '2010' AS year, STATEFP10 AS state, COUNTYFP10 AS county, GEOID10 AS precinct, 'polygon' AS accuracy FROM tl_2012_18_vtd10 WHERE COUNTYFP10 != '157' AND COUNTYFP10 != '097'" \
+	ogr2ogr -sql "SELECT '2010' AS year, STATEFP10 AS state, COUNTYFP10 AS county, GEOID10 AS precinct, 'polygon' AS accuracy FROM tl_2012_18_vtd10 WHERE COUNTYFP10 != '003' AND COUNTYFP10 != '057' AND COUNTYFP10 != '097' AND COUNTYFP10 != '157'" \
```
Please convert this to a alphabetical NOT IN list.

nvkelso commented:
Please also add/modify the README.md file in the indiana dir explaining the sources (especially how you might have scraped the Esri endpoints for the GeoJSON).

migurski commented:
Preview for commit 93dcb0dd2f61c5a7ea31baa02c35721c8e0bbf14:

![](https://s3.amazonaws.com/nvkelso-election-geodata/commits/93dcb0dd2f61c5a7ea31baa02c35721c8e0bbf14/render.png)

nvkelso commented:
About 30 minutes for the preview image to show up, not bad!

migurski commented:
Yeah, with most of the files already checked-out the actual processing doesn't take too long!

migurski commented:
Preview for commit 62008572bf9ac92890b7cfae324a9639a03774e5:

![](https://s3.amazonaws.com/nvkelso-election-geodata/commits/62008572bf9ac92890b7cfae324a9639a03774e5/render.png)

nvkelso commented:
Now just for Arizona, Missouri, Alabama, and New York and everything will
be connected, lol.

On Thu, Sep 14, 2017 at 1:17 PM, Michal Migurski <notifications@github.com>
wrote:

> Preview for commit 6200857
> <https://github.com/nvkelso/election-geodata/commit/62008572bf9ac92890b7cfae324a9639a03774e5>
> :
>
>
> <https://camo.githubusercontent.com/fce32581041a4695285e4b0063e49d9b3fe5cc90/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6e766b656c736f2d656c656374696f6e2d67656f646174612f636f6d6d6974732f363230303835373262663961633932383930623763666165333234613936333961303337373465352f72656e6465722e706e67>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nvkelso/election-geodata/pull/120#issuecomment-329596810>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0EO-pL1-h7AhG3k4-yUJwqgcJcj2O-ks5siYnRgaJpZM4PX3Pp>
> .
>

migurski commented:
Ha ha yeah, easy.

I’m looking at more of these counties by population. Lake County (Gary, IN) is a big one seemingly without shapefiles or an ArcGIS server, but [they do have PDF maps available](https://www.lakecountyin.org/portal/media-type/html/group/voters/page/default.psml/js_pane/P-144513c72a8-22548). 🤔

nvkelso commented:
The following link worked better for me:

-
https://www.lakecountyin.org/portal/media-type/html/group/voters/page/default.psml/js_pane/P-144513c72a8-22548

It does say GIS data, probably worth a quick email to them asking for the
source.


On Thu, Sep 14, 2017 at 1:21 PM, Michal Migurski <notifications@github.com>
wrote:

> Ha ha yeah, easy.
>
> I’m looking at more of these counties by population. Lake County (Gary,
> IN) is a big one seemingly without shapefiles or an ArcGIS server, but they
> do have PDF maps available
> <https://www.lakecountyin.org/portal/media-type/html/group/voters/page/default.psml/js_pane/P-144513c72a8-22548>.
> 🤔
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nvkelso/election-geodata/pull/120#issuecomment-329597940>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0EOwfK-QTywM-1o1T1bhtHYaqBLIryks5siYrTgaJpZM4PX3Pp>
> .
>

migurski commented:
I sent them a friendly email to see if they publish SHP or ArcGIS anywhere.

nvkelso commented:
Gah, GitHub keeps holding individual comments until I've submitted a formal review :\

migurski commented:
Preview for commit a62ecd68f73f2aa3dfeaaa1806966bd679a89468:

![](https://s3.amazonaws.com/nvkelso-election-geodata/commits/a62ecd68f73f2aa3dfeaaa1806966bd679a89468/render.png)

migurski commented:
I think I’ve addressed your comments. I’d like to keep this PR open a little while, to see if Lake county responds and to add a few more data sources.

nvkelso commented:
@migurski You still want to keep this PR open? Let's set a deadline of this Friday to merge it. Further work can be performed in a new PR...

migurski commented:
That works for me! There are a solid set of changes here that I’d be happy to merge.
